modules/registry-configuring-storage-nutanix.adoc

```diff
@@ -0,0 +1,9 @@
+// * installing/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-storage-nutanix_{context}"]
+= Configuring the default storage container
+
+After you install the cluster, you must install the Nutanix CSI Operator and configure the default storage container for the cluster.
```

Comment:
Is the Nutanix Operator supported by Red Hat? If not, do we need to include a note block stating this?

Reply:
Yes it is supported, but must be installed separately post-installation. For the time being the guidance from the team was to simply xref the Nutanix doc.